setup.cfg

```diff
@@ -42,7 +42,8 @@ install_requires =
 [options.extras_require]
 # The list of available extras is also in the installation docs, if you add or remove one please edit it there as well.
 asdf =
-  asdf>=2.6.0,<=3.0.0
+  asdf>=2.8.0
+  asdf-astropy>=0.1.1
```

Comment:
This new dependency should get a changelog entry, as should the bumped asdf requirement.